test/validate/eia_test.py

```diff
@@ -45,13 +45,13 @@ def test_no_null_cols_eia(pudl_out_eia, live_dbs, cols, df_name):
 @pytest.mark.parametrize(
     "df_name,raw_rows,monthly_rows,annual_rows",
     [
-        ("bf_eia923", 1_607_752, 1_607_752, 128_252),
+        ("bf_eia923", 1_607_692, 1_607_692, 128_252),
```

Comment:
There's hardly any change in the number of rows in this table. Does that seem correct? Why would it be such a small change?  And a decrease in rows, even though we're adding new data.

Reply:
I looked into this because I was also confused. It seems like the majority of the changes to this table where filling in previously NA records and then removing some NA records. I'm not exactly sure why they got removed, but they did. But this is happening in the raw data, so it's nothing we are doing in PUDL.

Reply:
Huh, okay that's odd. But if the new data really does run all the way through November I guess it's just a coincidence that it's almost exactly the same number of rows?